app/components/patron_field_component.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="row m-2">
```

Comment:
This is just here because I wanted to use the generic view component template, but didn't need to modify any of the parent class, just the template.